src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -201,6 +204,52 @@
 //%DllMap%
 }
 
+int32_t GlobalizationNative_LoadICUData(void *pData);
+
+static int32_t load_icu_data ()
+{
+    char path [1024];
+    int res;
+
+    const char *dname = "icudt.dat";
+    const char *bundle = get_bundle_path ();
+
+    os_log_info (OS_LOG_DEFAULT, "Loading ICU data file '%s'.", dname);
+    res = snprintf (path, sizeof (path) - 1, "%s/%s", bundle, dname);
+    assert (res > 0);
+
+    FILE *fp = fopen (path, "rb");
+    if (fp != NULL) {
+        if (fseek (fp, 0L, SEEK_END) != 0) {
+            os_log_info (OS_LOG_DEFAULT, "Unable to determine size of the dat file");
+            exit(1);
+        }
+
+        long bufsize = ftell (fp);
```

Comment:
I suppose we should check for errors here too – `ftell` will return -1 if the call fails.